core/controllers/dashboard.py

```diff
@@ -204,6 +204,7 @@ def _round_average_ratings(rating):
                     'created_on': utils.get_time_in_millisecs(
                         collection_summary.collection_model_created_on),
                     'status': collection_summary.status,
+                    'node_count': collection_summary.node_count,
```

Comment:
Actually, I'll not be able to work on this over the coming few days and even till the end of the next week (going on a vacation + have to attend Pycon India + college work).

I'm fine both ways - either keeping this PR open or creating a new PR for the pupose of resolution of that TODO. (Though my previous comment was based on the thinking that this PR could get merged asap, and then I'll open up a new one for just the purpose of resolving that TODO)
What do you suggest ?